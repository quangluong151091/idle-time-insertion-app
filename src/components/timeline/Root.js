import React from 'react';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';
import reduxPromise from 'redux-promise';
import reducers from './reducers';
import thunk from 'redux-thunk';

export default ({ children, initialState = {} }) => {
  const store = createStore(
    reducers,
    initialState,
    applyMiddleware(thunk,reduxPromise),
  );

  return <Provider store={store}>{children}</Provider>;
};
