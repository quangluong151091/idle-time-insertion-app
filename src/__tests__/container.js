import React from 'react';
import { shallow } from 'enzyme';

import Root from '../components/timeline/Root';
import Timeline from '../components/timeline/timeline/containers';

describe('TimelineContainer', () => {
  it('should render correctly', () => {

    const component = shallow(<Root><Timeline /></Root>);
  
    expect(component).toMatchSnapshot();
  });
});