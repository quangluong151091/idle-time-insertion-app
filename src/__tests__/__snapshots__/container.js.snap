// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineContainer should render correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(timelineLayout) />
</Provider>
`;
